code/modules/clothing/head/misc.dm

```diff
@@ -327,6 +327,10 @@
 	item_state = "chickensuit"
 	flags = BLOCKHAIR
 
+/obj/item/clothing/head/corgi/en
+	name = "E-N suit head"
+	icon_state = "enhead"
+
```

Comment:
This seems to be already defined under a different path:

![image](https://user-images.githubusercontent.com/42044220/203220744-67331632-0e28-43b4-9ca5-42258d1c2e6b.png)